CODE-OF-CONDUCT.md

```diff
@@ -44,7 +44,7 @@ response to any behavior that they deem inappropriate, threatening, offensive,
 or harmful.
 
 Community leaders have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
+comments, commits, code, documentation edits, issues, and other contributions that are
```

Comment:
This is a good change, but we should be aware that the Code of Conduct is taken as-is from https://www.contributor-covenant.org/version/2/1/code_of_conduct. So there's a chance it'll get reverted when a newer version comes out.